spec/burndown_data_spec.rb

```diff
@@ -5,183 +5,64 @@
-      @burndown.extra_tasks.open = 7
-      @burndown.extra_tasks.done = 8
-      @burndown.date_time = DateTime.parse("2015-01-15")
-
-      expected_hash = {
-        "date" => "2015-01-15",
-        "story_points" => {
-          "total" => 3,
-          "open" => 1
-        },
-        "tasks" => {
-          "total" => 7,
-          "open" => 3
-        },
-        "story_points_extra" => {
-          "done" => 6
-        },
-        "tasks_extra" => {
-          "done" => 8
-        }
-      }
-
-      expect(@burndown.to_hash).to eq(expected_hash)
-    end
-  end
-
-  describe "#find_list_by_title" do
-    before(:each) do
-      list_url_match = /https:\/\/trello.com\/1\/boards\/myboardid\/lists\?-*/
-
-      stub_request(:any,list_url_match).to_return(:status => 200,
-        :body => load_test_file("lists.json"), :headers => {})
-    end
-
-    it "returns list id for title matching string" do
-      expect( @burndown.find_list_by_title("Done Sprint 9") ).
-        to eq "5319bf045c6ef0092c55331e"
-    end
-
-    it "doesn't return list id if title doesn't match string" do
-      expect( @burndown.find_list_by_title("Sprint 9") ).
-        to be(nil)
-    end
-
-    it "returns list id for title matching regexp" do
-      expect( @burndown.find_list_by_title(/Sprint 9$/) ).
-        to eq "5319bf045c6ef0092c55331e"
-    end
-  end
-
-  describe "#fetch_todo_list_id" do
-    it "returns list id" do
-      list_url_match = /https:\/\/trello.com\/1\/boards\/myboardid\/lists\?-*/
-          
-      stub_request(:any,list_url_match).to_return(:status => 200,
-        :body => load_test_file("lists.json"), :headers => {})
-
-      expect( @burndown.fetch_todo_list_id ).to eq "53186e8391ef8671265eba9e"
-    end
-    
-    it "raises exception if it does not find done column" do
-      expect{ @burndown.fetch_todo_list_id }.to raise_error
-    end
-  end
-  
-  describe "#fetch_doing_list_id" do
-    it "returns list id" do
-      list_url_match = /https:\/\/trello.com\/1\/boards\/myboardid\/lists\?-*/
-          
-      stub_request(:any,list_url_match).to_return(:status => 200,
-        :body => load_test_file("lists.json"), :headers => {})
-
-      expect( @burndown.fetch_doing_list_id ).to eq "53186e8391ef8671265eba9f"
-    end
-    
-    it "raises exception if it does not find done column" do
-      expect{ @burndown.fetch_doing_list_id }.to raise_error
+      expect(@burndown.tasks.open).to eq 42
     end
   end
-  
-  describe "#fetch_done_list_id" do
-    it "returns list id" do
-      list_url_match = /https:\/\/trello.com\/1\/boards\/myboardid\/lists\?-*/
-          
-      stub_request(:any,list_url_match).to_return(:status => 200,
-        :body => load_test_file("lists.json"), :headers => {})
 
-      expect( @burndown.fetch_done_list_id ).to eq "5319bf088cdf9cd82be336b0"
-    end
-    
-    it "raises exception if it does not find done column" do
-      expect{ @burndown.fetch_done_list_id }.to raise_error
-    end
```

Comment:
It would be good to keep the tests which check the error cases. Or did I just miss them being at another place?